configure.ac

```diff
@@ -135,6 +135,9 @@ AC_ARG_WITH([systemdsystemunitdir],
 AC_ARG_ENABLE(debug,
               [  --enable-debug      Enable debugging code.],, enable_debug="no")
 
+AC_ARG_ENABLE(extra-warnings,
+              [  --disable-extra-warnings      Disable extra compiler warnings.],, enable_extra_warnings="yes")
+
 AC_ARG_ENABLE(env-wrapper,
```

Comment:
I don't see why we need this switch, but I doesn't bother me too much either. It defaults to "yes" and I doubt anyone would ever disable this explicitly.